### ISSUE TITLE
Invalid 'group': 'lspconfig'

### ISSUE THREAD

**Issue by wert2all** [AstroNvim/AstroNvim#2524]
### Checklist

- [X] I have searched through the AstroNvim docs
- [X] I have searched through the existing issues of AstroNvim
- [X] I have searched the existing issues of plugins related to this issue

### Neovim version (nvim -v)

NVIM v0.9.5

### Operating system/version

openSUSE Tumbleweed & arch

### Terminal/GUI

alacritty

### Describe the bug

Error on clean install

> Error executing vim.schedule lua callback: .../.local/share/nvim/lazy/astrocore/lua/astrocore/init.lua:106: Invalid 'group': 'lspconfig'
stack traceback:
        [C]: in function 'nvim_exec_autocmds'
        .../.local/share/nvim/lazy/astrocore/lua/astrocore/init.lua:106: in function 'exec_buffer_autocmds'
        ...zy/AstroNvim/lua/astronvim/plugins/configs/lspconfig.lua:4: in function <...zy/AstroNvim/lua/astronvim/plugins/configs/lspconfig.lua:2>
Press ENTER or type command to continue

### Steps to Reproduce

Clean install AstroVim
1.  clone template
2. rm .git
3. nvim

### Expected behavior

Nvim should start normally

### Repro

```Lua
-- save as repro.lua
-- run with nvim -u repro.lua
-- DO NOT change the paths
local root = vim.fn.fnamemodify("./.repro", ":p")

-- set stdpaths to use .repro
for _, name in ipairs({ "config", "data", "state", "runtime", "cache" }) do
  vim.env[("XDG_%s_HOME"):format(name:upper())] = root .. "/" .. name
end

-- bootstrap lazy
local lazypath = root .. "/plugins/lazy.nvim"
if not vim.loop.fs_stat(lazypath) then
  -- stylua: ignore
  vim.fn.system({ "git", "clone", "--filter=blob:none", "https://github.com/folke/lazy.nvim.git", "--branch=stable", lazypath })
end
vim.opt.rtp:prepend(vim.env.LAZY or lazypath)

-- install plugins
local plugins = {
  { "AstroNvim/AstroNvim", branch = "v4", import = "astronvim.plugins" },

  -- add any other plugins/customizations here
}
require("lazy").setup(plugins, {
  root = root .. "/plugins",
}):wqa

-- add anything else here (autocommands, vim.filetype, etc.)
```

**Comment by mehalter** [comment]
Thanks for reporting! This is resolved in AstroCore: https://github.com/AstroNvim/astrocore/commit/1736458b321c59c6e57c456f0f39c0665b301591

Release will be tagged soon and I will close this

**Comment by mehalter** [comment]
Ok, this should be resolved in AstorCore v1.1.1 ! Update with `:Lazy update` thanks again!

**Comment by wert2all** [comment]
Thanks for your quick answer!
